pyresample/future/spherical/arc.py

```diff
@@ -34,9 +34,11 @@ def _check_valid_arc(start_point, end_point):
         raise ValueError("An SArc can not be represented by the same start and end SPoint.")
     if start_point.is_pole() and end_point.is_pole():
         raise ValueError("An SArc can not be uniquely defined between the two poles.")
-    if start_point.is_on_equator and end_point.is_on_equator and abs(start_point.lon - end_point.lon) == np.pi:
+    if start_point.is_on_equator() and end_point.is_on_equator() and abs(start_point.lon - end_point.lon) == np.pi:
         raise ValueError(
             "An SArc can not be uniquely defined on the equator if start and end points are 180 degrees apart.")
+    if start_point.get_antipode() == end_point:
```

Comment:
Similar to the `is_on_equator` should this just be a `antipode` property? Possibly cached if it is used a lot.